toolkit/scalebar/src/main/java/com/arcgismaps/toolkit/scalebar/Scalebar.kt

```diff
@@ -66,12 +66,7 @@ public fun Scalebar(
     minScale: Double = 0.0, // minimum scale to show the scalebar
     useGeodeticCalculations: Boolean = true, // `false` to compute scale without a geodesic curve,
     style: ScalebarStyle = ScalebarStyle.AlternatingBar,
-    // TODO: determining the default ScalebarUnit is not tested
-    units: ScalebarUnits = if (isMetric()) {
-        ScalebarUnits.METRIC
-    } else {
-        ScalebarUnits.IMPERIAL
-    },
+    units: ScalebarUnits = rememberDefaultScalebarUnits(),
```

Comment:
I don't think we need a new class to represent the Units, 
we can use [com.arcgis.UnitSystem](https://developers.arcgis.com/kotlin/api-reference/arcgis-maps-kotlin/com.arcgismaps/-unit-system/index.html), which is already serializable and used across the arcgis-maps SDK
```suggestion
    units: UnitSystem = rememberDefaultScalebarUnits(),
```